docs/internals/data-structures.rst

```diff
@@ -233,8 +233,11 @@ state).
 .. rubric:: Compatibility with older servers and enabling quota after-the-fact
 
 If no quota data is stored in the hints file, Borg assumes zero quota is used.
-Thus, if a repository with an enabled quota is written to with an older version
-that does not understand quotas, then the quota usage will be erased.
+Thus, if a repository with an enabled quota is written to with an older ``borg serve``
```

Comment:
Same problem happens when a older borg accesses a locally mounted repository (not ssh: repo url).